superduperdb/base/config.py

```diff
@@ -138,29 +142,29 @@ class Config(JSONable.Config):
 
     @property
     def comparables(self):
-        '''
+        """
```

Comment:
lets create a check or lint which can convert all 
three quote into single quotes
its standard in python to either keep single or double quotes for comment and it should be consist with the codebase